visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -628,7 +628,31 @@
              </widget>
             </item>
             <item>
-             <widget class="QDoubleSpinBox" name="exec_time_factor">
+             <widget class="QDoubleSpinBox" name="velocity_scaling_factor">
+              <property name="maximum">
+               <double>1.000000000000000</double>
+              </property>
+              <property name="singleStep">
+               <double>0.010000000000000</double>
+              </property>
+              <property name="value">
+               <double>1.000000000000000</double>
+              </property>
+             </widget>
+            </item>
+           </layout>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_20">
+            <item>
+             <widget class="QLabel" name="label_20">
+              <property name="text">
+               <string>Acceleration Scaling:</string>
```

Comment:
Before and after.  No change in width but the widget did get one line taller (expected).

![screenshot from 2016-01-14 11 29 47](https://cloud.githubusercontent.com/assets/9295956/12332181/af0b5730-bab2-11e5-9253-68a833adfa39.png)

![After](https://cloud.githubusercontent.com/assets/9295956/12332143/83ead986-bab2-11e5-9a44-d7e26386452d.png)